exovetter/utils.py

```diff
@@ -0,0 +1,423 @@
+"""Utility functions."""
+
+import sys
+import warnings
+
+import numpy as np
+
+__all__ = ['sine', 'mark_transit_cadences', 'WqedLSF']
```

Comment:
Any public functions that should be documented for user needs to be added to `__all__` too. If a function is intended for internal use only, consider prefixing with one underscore in the function name.